### DISCUSSION TITLE
[ FF7 ] 7th Game Crashes Upon Entering Any Town

### DISCUSSION THREAD

**Discussion by Shinlowkey** [julianxhokaxhiu/FFNx#190] [General]
Hello, as the title stated, my game recently crashed today upon entering any town. I played for 20 hours with no problems but I decided to go inside the 7th Heaven application and tweak  the mod settings and reconfigured some of them to REMAKO settings. After experiencing the ninostyle for the first time, it was great. Everything was normal during battles but entering a town would crash the game. It gave me the OOPS error notification then asked me to copy the error and put it here but I'm not sure which application to use because the letters look funky in notepad. I tried redownloading the game and validating the files but none of that seem to fix the problem. I deactivated all of the mods and it worked. I'm just not sure which mod is crashing my game. If there's any help I can get I would greatly appreciated. Thank you!

-Shinlowkey

**Comment by julianxhokaxhiu** [comment]
Hi thanks for having me approached.

If you're using 7h please make sure you approach the 7h author despite the crash. The support channel here is only if you use FFNx outside of 7h.

Best regards.
